src/Backend/API.ts

```diff
@@ -35,47 +34,53 @@ let globalBackendMap: BackendMap = {};
 // List of Executor extensions registered
 let globalExecutorArray: Executor[] = [];
 
-function backendRegistrationApi() {
-  const logTag = "backendRegistrationApi";
-  let registrationAPI = {
-    registerBackend(backend: Backend) {
-      const backendName = backend.name();
-      assert(backendName.length > 0);
-      globalBackendMap[backendName] = backend;
-      const compiler = backend.compiler();
-      if (compiler) {
-        gToolchainEnvMap[backend.name()] = new ToolchainEnv(compiler);
-      }
-      const executor = backend.executor();
-      if (executor) {
-        globalExecutorArray.push(executor);
-      }
-      Logger.info(
-        logTag,
-        "Backend",
-        backendName,
-        "was registered into ONE-vscode."
-      );
-      // NOTE: This might not 100% guaratee the activating extension has been done.
-      //   - link: https://github.com/Samsung/ONE-vscode/pull/1101#issuecomment-1195099002
-      // TODO: Consider better way to refresh toolchainView after backend's registration.
-      vscode.commands.executeCommand("one.toolchain.refresh");
-      vscode.commands.executeCommand("one.device.refresh");
-    },
-    registerExecutor(executor: Executor) {
-      globalExecutorArray.push(executor);
-      Logger.info(
-        logTag,
-        "Executor",
-        executor.name(),
-        "was registered into ONE-vscode."
-      );
-    },
-  };
+const logTag = "API";
 
-  registrationAPI.registerBackend(new OneToolchain());
+const registerBackend = (backend: Backend) => {
+  const backendName = backend.name();
+  assert(backendName.length > 0);
+  globalBackendMap[backendName] = backend;
+  const compiler = backend.compiler();
+  if (compiler) {
+    gToolchainEnvMap[backend.name()] = new ToolchainEnv(compiler);
+  }
+  const executor = backend.executor();
+  if (executor) {
+    globalExecutorArray.push(executor);
+  }
 
-  return registrationAPI;
-}
+  Logger.info(
+    logTag,
+    "Backend",
+    backendName,
+    "was registered into ONE-vscode."
+  );
+
+  // NOTE: This might not 100% guaratee the activating extension has been done.
+  //   - link: https://github.com/Samsung/ONE-vscode/pull/1101#issuecomment-1195099002
+  // TODO: Consider better way to refresh toolchainView after backend's registration.
+  vscode.commands.executeCommand("one.toolchain.refresh");
+  vscode.commands.executeCommand("one.device.refresh");
+};
+
+const registerExecutor = (executor: Executor) => {
+  globalExecutorArray.push(executor);
+  Logger.info(
+    "API",
+    "Executor",
+    executor.name(),
+    "was registered into ONE-vscode."
+  );
+
+  // NOTE: This might not 100% guaratee the activating extension has been done.
+  //   - link: https://github.com/Samsung/ONE-vscode/pull/1101#issuecomment-1195099002
+  // TODO: Consider better way to refresh toolchainView after backend's registration.
+  vscode.commands.executeCommand("one.device.refresh");
+};
```

Comment:
I don't know whether we need this function now. However, this PR just changes the code structure. Let's think about how to do this function later.

Reply:
I agree. I thought about removing this one because it's not used anywhere. I left this one in this PR but I agree that we clean this up